src/ophys_etl/transforms/suite2p_wrapper.py

```diff
@@ -0,0 +1,249 @@
+                         "scalar multiplier. From an extensive validation "
+                         "effort, we found that 0.75 segmented expected cells "
+                         "but higher values tended to miss some. A fair "
+                         "number of non-cells also are segmented, and need "
+                         "to be filtered by a classifier."))
+    max_overlap = argschema.fields.Float(
+            default=0.75,
+            description=("Cells with more overlap than this get removed "
+                         "during triage, before refinement. Suite2P default."))
+    high_pass = argschema.fields.Int(
+            default=100,
+            description=("Running mean subtraction with window of "
+                         "size 'high_pass'. Suite2P default."))
+    smooth_masks = argschema.fields.Bool(
+            default=True,
+            description=("Whether to smooth masks in final pass of cell "
+                         "detection. Especially useful if one is in a high "
+                         "noise regime. It appears this is only applicable "
+                         "with sparse_mode=False, which is not our default."))
+    # s2p ROI extraction parameters
+    inner_neuropil_radius = argschema.fields.Int(
+            default=2,
+            description=("Number of pixels to keep between ROI and neuropil "
+                         "donut. "
+                         "Allen production will run it's own neuropil "
+                         "subtraction routine."))
+    min_neuropil_pixels = argschema.fields.Int(
+            default=350,
+            description=("Minimum number of pixels in the neuropil."
+                         "Allen production will run it's own neuropil "
+                         "subtraction routine."))
+    allow_overlap = argschema.fields.Bool(
+            # NOTE: changing this from extraction validation and labeling
+            # will be untangled downstream in demixing.
+            default=True,
+            description=("Pixels that are overlapping are thrown out (False) "
+                         "or added to both ROIs (True). Allen production will "
+                         "run a demixing step that should allow both ROIs "
+                         "to share pixels."))
+    # Allen-specific options
+    bin_size = argschema.fields.Int(
+            required=False,
+            description=("If nbinned not provided, will calculate nbinned as "
+                         "nframes / bin_size. This allows consistent temporal "
+                         "downsampling across movies of different lengths."))
+    output_dir = argschema.fields.OutputDir(
+            required=True,
+            description="for minimal and cleaner output, specifies output dir")
+    retain_files = argschema.fields.List(
+            argschema.fields.Str,
+            cli_as_single_argument=True,
+            required=True,
+            default=['stat.npy'],
```

Comment:
Just double checking since I've forgotten everything about Suite2P output files but we don't need to retain stuff like `F.npy` or `Fneu.npy`?

Reply:
adding those to this list will preserve them. We can change the default later as feature engineering proceeds, if we want Suite2P's traces. `stat.npy` is saving just weighted masks.